Code/GraphMol/TautomerQuery/TautomerQuery.cpp

```diff
@@ -243,7 +241,7 @@ std::vector<MatchVectType> TautomerQuery::substructOf(
   templateParams.extraFinalCheck = checker;
 
   auto matches =
-      RDKit::SubstructMatch(mol, *d_templateMolecule, templateParams);
+      RDKit::SubstructMatch(mol, *d_templateMolecule.get(), templateParams);

```

Comment:
I don't think you need the `.get()` here:
```suggestion
      RDKit::SubstructMatch(mol, *d_templateMolecule, templateParams);

```